Client/Frontend/Strings.swift

```diff
@@ -184,3 +184,8 @@ extension Strings {
     public static let UnableToAddPassErrorMessage = NSLocalizedString("AddPass.Error.Message", value: "An error occured while adding the pass to Passbook. Please try again later.", comment: "Text of the 'Add Pass Failed' alert")
     public static let UnableToAddPassErrorDismiss = NSLocalizedString("AddPass.Error.Dismiss", value: "OK", comment: "Button to dismiss the 'Add Pass Failed' alert")
 }
+
+// Top Tabs
+extension Strings {
+    public static let TopTabsNewTabTitle = NSLocalizedString("TopTabs.Tiltle.NewTab", value: "New Tab", comment: "The title displayed in the tab when opening a new tab.")
```

Comment:
I would keep this string where it was instead of creating a new localization value for it since it will appear as a new string for the localizers. Also theres a type in TopTabs.Tiltle.NewTab.